src/configParamsJSON/dataSliderConfigParams.json

```diff
@@ -291,7 +291,19 @@
                       "defaultValue": "Okay"
                     }
                   ]
-                },
+                }
```

Comment:
Looks like maybe just formatting changes? But wanted to make sure you meant to include these?

Reply:
Forgot to link the issue - It's for this: https://devtopia.esri.com/WebGIS/arcgis-template-configuration/issues/3048